code/tic-tac-tow-service/src/main/kotlin/pt/isel/daw/tictactow/TicTacTowApplication.kt

```diff
@@ -19,11 +22,24 @@ import java.time.Instant
 @SpringBootApplication
 class TicTacTowApplication {
     @Bean
-    fun jdbi() = Jdbi.create(
-        PGSimpleDataSource().apply {
-            setURL("jdbc:postgresql://localhost:5432/db?user=dbuser&password=changeit")
+    fun jdbi(): Jdbi {
+        val postgresUri =
+            System.getenv("POSTGRES_URI")
+                ?: "jdbc:postgresql://localhost:5432/db?user=dbuser&password=changeit"
```

Comment:
we should not have connections URIs hardcoded in the code, specially passwords.

Reply:
Agree. Please remove this hardcoded URL